discovery-provider/default_config.ini

```diff
@@ -13,6 +13,7 @@ identity_service_url = https://identityservice.test
 user_metadata_service_url = ''
 healthy_block_diff = 100
 notifications_max_block_diff = 25
+default_oracle_address = 0xFakePublicKey
```

Comment:
isn't this fake pub key not a real hex string? - maybe it doesn't matter 🤷

Reply:
for now doesn't matter since we're not doing anything with it